script.js

```diff
@@ -0,0 +1,78 @@
+const books = [];
```

Comment:
- [ ] Please make sure you have used JavaScript classes instead of objects and arrays, you can use objects and arrays but it has to be inside the classes.